src/Language/PureScript/TypeChecker/Types.hs

```diff
@@ -5,6 +5,7 @@
 --
 module Language.PureScript.TypeChecker.Types
   ( BindingGroupType(..)
+  , TypedValue'(..)
```

Comment:
Last thing, I promise: does this need to be exported? I'd prefer not to if we can avoid it.

Reply:
No worries, and yeah that's a good catch. In my earlier iterations I had needed to export it but no longer :)